crates/sui-graphql-rpc/src/extensions/query_limits_checker.rs

```diff
@@ -326,34 +336,60 @@ impl QueryLimitsChecker {
     }
 }
 
-fn check_limits(limits: &Limits, cost: &ComponentCost, pos: Option<Pos>) -> ServerResult<()> {
+fn check_limits(
+    limits: &Limits,
+    cost: &ComponentCost,
+    pos: Option<Pos>,
+    ctx: &ExtensionContext<'_>,
+) -> ServerResult<()> {
+    let query_id = ctx.data_unchecked::<Uuid>().to_string();
+    let session_id = ctx.data_unchecked::<SocketAddr>().to_string();
```

Comment:
(ditto)

Reply:
Same as before, it will not work for the query_id and session_id variables as tracing does not seem to be able to convert Uuid into string.